### PR TITLE
update import script for edge cases

### LINKED ISSUE #316: Update data import script to account for edge cases
- Update script to account for 164 users under various edge cases who currently have two entries (i.e. their hash appears twice in the final data), based on EDD's upcoming response 
- There's a separate automatic id field for now, but once the hashes are unique, that should be renamed to id so we can set that as the partition key and disable the automatic id creation. https://docs.microsoft.com/en-us/azure/cosmos-db/partitioning-overview (edited) 

-------
- Import updated data file that uses DOB instead of eddcan (shouldn't require any changes in our db logic)
- Update import logic to form an array of seekWorkPlans and programs
- If a user has the same week listed twice, only import their PUA entry

### PR DESCRIPTION
**Pull request by kalvinwang**
This PR updates the import script to account for edge cases, and processes the new variety of SeekWorkPlans we're receiving in the updated data file.

Each element of the output has the following properties: id (the hash), programSeekWorkPlan (PUA, UI full time, or UI part time; or an array of these three values), and weeksToCertify (an array of integer indices corresponding to possible weeks)

===

Resolves #316

- [ ] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [ ] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

### CODE REVIEW

**Comment by kalvinwang** [comment]
Still waiting for Alex to get back to us on whether "T" is UI full-time or part-time, but this is otherwise ready.

File: scripts/import_users.py [hunk 1]
```diff
@@ -66,45 +72,108 @@ def print_total_weeks_by_user(df):
...
-        # df.pivot_table(index=["SHA256_hash"], aggfunc="size").sort_values()
         df["WeekEndingDates"] = df["WeekEndingDates"].apply(lambda x: [INDEX_TO_WEEKS[y] for y in x])
         dupe_hashes = df[df.duplicated("SHA256_hash", keep=False)]
         dupe_hashes_count = len(dupe_hashes)
         print(f"There are {dupe_hashes_count} duplicate hashes:")
         print(dupe_hashes)
         dupe_hashes.to_excel(DUPLICATE_HASHES_FILENAME)
 
+    # row2 will be prioritized if duplicate found, so make sure row2, and not row1, is the PUA row
+    def insert_sorted(row1, row2, first_hashes, is_both_ui):
```
Comment (sharonwarner) [inline scripts/import_users.py:83]: Could we name row2 'pauRow' instead?

File: scripts/import_users.py [hunk 2]
```diff
@@ -66,45 +72,108 @@ def print_total_weeks_by_user(df):
...
+        pua_row = user_rows[mask]
+        ui_rows = user_rows[~mask] # apply inverse of mask
+        if len(pua_row) == 1:
+            assert(len(ui_rows) == 1)
+            # there's one PUA entry and one UI, merge in UI row weeks, making sure PUA overrides UI if same week
+            insert_sorted(ui_rows, pua_row, first_hashes, False)
+        else:
+            assert (len(ui_rows) == 2)
+            # there's two UI entries (UI part time and UI full time), merge and check no overlap on the same week
+            insert_sorted(ui_rows.iloc[[0]], ui_rows.iloc[[1]], first_hashes, True)
```
Comment (sharonwarner) [inline scripts/import_users.py:131]: 👍

File: scripts/import_users.py (hunk 1 above)
Comment (kalvinwang) [reply scripts/import_users.py:83]: no, it gets called in two places, and in one scenario it's the other UI row (part time vs full time)... but i'll try to make that more obvious in the code or comments

**Pull request merged**